portfolio/src/main/webapp/js/custom-script.js

```diff
@@ -43,6 +43,7 @@ const MNPLS_LNG = -93.2650;
 
 function onLoad() {
```

Comment:
so this onLoad() function will be for the search bar on the home page? Just to differentiate like statisticsOnLoad(), I'd name this something similar like homeOnLoad().

Reply:
It is for the home page and the about page.